.github/workflows/release-test.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         with:
           distribution: goreleaser-pro
           version: "v2.8.2"
-          args: release --skip=publish --skip=validate --skip=sign --clean --snapshot
+          args: release --skip=publish --skip=validate --skip=sign --clean
```

Comment:
--snapshot adds to the tag name. I think its better if the output version matches the input version when the user kicks off the workflow.